psutil/arch/windows/process_info.h

```diff
@@ -15,3 +20,7 @@ int psutil_pid_in_proclist(DWORD pid);
 int psutil_pid_is_running(DWORD pid);
 PVOID psutil_get_peb_address(HANDLE ProcessHandle);
 PyObject* psutil_get_arg_list(long pid);
+int psutil_get_proc_info(DWORD pid, PSYSTEM_PROCESS_INFORMATION *retProcess,
+                 PVOID *retBuffer);
```

Comment:
Please indent "PVOID *retBuffer);" so that it matches the parenthesis "(" above.